src/main/java/com/josdem/vetlog/model/Pet.java

```diff
@@ -14,84 +14,80 @@
 limitations under the License.
  */
 
-package com.josdem.vetlog.model;
-
-import static jakarta.persistence.EnumType.STRING;
-import static jakarta.persistence.GenerationType.IDENTITY;
-
-import com.josdem.vetlog.enums.PetStatus;
-import jakarta.persistence.CascadeType;
```

Comment:
Please do not format the code, it makes harder to review your changes, we have spotless plugin to format the code automatically when we run tests, for more information see: https://github.com/josdem/vetlog-spring-boot/blob/559f732d932ca49c4452153039161d7c9073f45f/build.gradle#L124